### ISSUE TITLE
Updating passwords visual bug

### ISSUE THREAD

**Issue by allanabiud** [keepassxreboot/keepassxc#9972]
## Overview
When trying to update passwords for entries after changing the password on the website and clicking the update button on the KeePassXC browser extension banner, I am not redirected to KeePassXC or a pop up does not show asking to confirm the update. 
Instead, I have to open KeePassXC manually and the pop up is there but it's not visually clear.

## Steps to Reproduce
1. Have the KeePassXC Browser extension connected and running.
2. Update the password of an already existing entry on a website.
3. Click on the update button on the update entry banner.
4. Note that you are not prompted to confirm update of the entry.
5. Manually open KeePassXC.
6. See bug.

## Expected Behavior
After updating a password and clicking on the update button, a KeePassXC prompt should appear asking for confirmation of this update.

## Actual Behavior
No prompt appears from KeePassXC and the prompt in KeePassXC is a bit obscured.

## Context
![KeePassXC_rehnIVc0m2](https://github.com/keepassxreboot/keepassxc/assets/66108188/f76aa97f-8bea-4558-a743-5467b3ac0534)

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - 2.7.6

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows 10

**Comment by droidmonkey** [comment]
That is not a bug I have seen before. Looks like the message prompt is melted into the main ui. Do you have any weird desktop tools running that would interfere with dialogs like that?

**Comment by varjolintu** [comment]
Might be related to this: https://github.com/keepassxreboot/keepassxc/pull/9853. Try if the latest snapshot build and see if the bug still exist.

**Comment by allanabiud** [comment]
> That is not a bug I have seen before. Looks like the message prompt is melted into the main ui. Do you have any weird desktop tools running that would interfere with dialogs like that?

Not really. After pressing the update button, the prompt doesn't appear for me. It appears when I manually open KeePassXC and even then it appears as shown in the image above.

**Comment by allanabiud** [comment]
> That is not a bug I have seen before. Looks like the message prompt is melted into the main ui. Do you have any weird desktop tools running that would interfere with dialogs like that?

And also, the password generator doesn't seem to be working in the latest KeePassXC 2.8.0 snapshot

**Comment by droidmonkey** [comment]
We know, that is going to be fixed today. Does the issue you described above still happen with the 2.8.0 snapshot?

**Comment by allanabiud** [comment]
> We know, that is going to be fixed today. Does the issue you described above still happen with the 2.8.0 snapshot?

Yes it still happens

**Comment by m-rz** [comment]
I've encountered this issue on my two PCs and then reproduced it on a clean Windows 10 VM with KeePassXC 2.7.6. Can confirm that the update prompt does not appear automatically and is rendered in a weird way - for sure not only OP is affected. Here's a video reference for this bug:

https://github.com/keepassxreboot/keepassxc/assets/9342509/8c92e9aa-a1c7-498f-9ea0-29b0b0f42437

**Comment by allanabiud** [comment]
> I've encountered this issue on my two PCs and then reproduced it on a clean Windows 10 VM with KeePassXC 2.7.6. Can confirm that the update prompt does not appear automatically and is rendered in a weird way - for sure not only OP is affected. Here's a video reference for this bug:
> 
>  Win10.Running.-.Oracle.VM.VirtualBox.2023-12-01.13-24-01.-.Trim.mp4

Thanks for sharing this. Shows it more clearly

**Comment by varjolintu** [comment]
This should be fixed with https://github.com/keepassxreboot/keepassxc/pull/9853. Test the latest snapshot build.
